packages/compass-editor/src/base-editor.tsx

```diff
@@ -70,6 +70,7 @@ export type EditorProps = {
 } & Omit<IAceEditorProps, 'onChange' | 'value' | 'theme'>;
 
 const editorStyle = css({
+  lineHeight: `${spacing[3]}px`,
```

Comment:
I am wondering what is the right format for such properties as lineHeight, fontSize, etc. I can see that in some places we use `lineHeight: 16` and in other `lineHeight: '20px'`. Same with fontSize: `fontSize: 12` vs fontSize: '14px'.